miden-lib/asm/miden/kernels/tx/asset.masm

```diff
@@ -3,28 +3,28 @@ use.miden::kernels::tx::account
 # ERRORS
 # =================================================================================================
 
-# The felt at position 1 must be zero
+# There is a malformatted asset, ASSET[1] must be 0
 const.ERR_FUNGIBLE_ASSET_FORMAT_POSITION_ONE_MUST_BE_ZERO=0x00020033
 
-# The felt at position 2 must be zero
+# There is a malformatted asset, ASSET[2] must be 0
 const.ERR_ASSET_FORMAT_POSITION_TWO_MUST_BE_ZERO=0x00020034
```

Comment:
Similar comment re description as above.

But also, should the name be `ERR_FUNGIBLE_ASSET_FORMAT_POSITION_TWO_MUST_BE_ZERO`?